### ISSUE TITLE
[BUG] NOT working version 1.18.1

### ISSUE THREAD

**Issue by ahbesuat** [MCCTeam/Minecraft-Console-Client#1953]
### Prerequisites

- [X] I made sure I am running the latest [development build](https://ci.appveyor.com/project/ORelio/minecraft-console-client/build/artifacts)
- [X] I tried to [look for similar issues](https://github.com/MCCTeam/Minecraft-Console-Client/issues?q=is%3Aissue) before opening a new one
- [X] I have set `debugmessages=true` in config to diagnose my issue
- [X] I have redacted session tokens and passwords before attaching screenshots

### Minecraft Version

1.18.1

### Console Client Version

latest

### Expected Behavior

I want to use oreminer version 1.18.1

### Actual Behavior

It's not working version 1.18.1 
[Warn] Update: Got error from scriptloader.script+oreminer: system.NullReferenceException

### Steps to Reproduce the bug

1. .
2. .
3. .

### Attach screenshot here (If applicable)

![image](https://user-images.githubusercontent.com/100497856/155863943-52544e26-4093-4373-827f-a801032b932b.png)

### Anythings that could help diagnosing the bug

```txt
//MCCScript 1.0

MCC.LoadBot(new OreMiner());

//MCCScript Extensions

/// <summary>
/// This bot can mine blocks that auto-spawn at given locations
/// </summary>
public class OreMiner: ChatBot
{
    // === CONFIG - REPLACE BLOCK LOCATION x y z VALUES HERE ===
    List<Location> location = new List<Location>()
    {
        //right
        new Location(-6021, 72, 19199),
        new Location(-6021, 72, 19198),
        new Location(-6021, 72, 19197),
	    new Location(-6021, 72, 19196),
	    new Location(-6021, 72, 19195),
        new Location(-6021, 72, 19205),
        new Location(-6021, 72, 19204),
        new Location(-6021, 72, 19203),
        new Location(-6021, 72, 19202),
        new Location(-6021, 72, 19201),
        //left
        new Location(-6023, 72, 19199),
        new Location(-6023, 72, 19198),
        new Location(-6023, 72, 19197),
	    new Location(-6023, 72, 19196),
	    new Location(-6023, 72, 19195),
        new Location(-6023, 72, 19205),
        new Location(-6023, 72, 19204),
        new Location(-6023, 72, 19203),
        new Location(-6023, 72, 19202),
        new Location(-6023, 72, 19201),
        // Add more her4
    };
    // === END OF CONFIG ===
    int index = 0;

    public override void Initialize()
    {
        LogToConsole("Bot enabled!");
    }

    public override void Update()
    {
        Material blockType = GetWorld().GetBlock(location[index]).Type;
        switch (blockType)
        {
            //Adjust here block types to mine
            case Material.Cobblestone:
	        case Material.DiamondOre:
            case Material.EmeraldOre:
            case Material.GoldOre:
            case Material.IronOre:
            case Material.CoalOre:
            case Material.LapisOre:
                DigBlock(location[index]);
                break;
        }
        index++;
        if (index >= location.Count)
            index = 0;
    }
}
```

### Device

Laptop

### Operating System

Windows

### Server Address (If applicable)

play.ruincraft.xyz

**Comment by ReinforceZwei** [comment]
`OreMiner` requires terrain and movement handling to be enabled but it is not yet implemented in 1.18.1.

**Comment by ahbesuat** [comment]
how can i do it or when can you do it

**Comment by farfromsubtlee** [comment]
@ahbesuat if your server supports older versions you can set the client version to 1.16.5.

**Comment by milutinke** [comment]
Fixed in #1943.
